android/BOINC/app/src/main/java/edu/berkeley/boinc/PrefsFragment.java

```diff
@@ -79,9 +78,10 @@ public void onReceive(Context context,Intent intent) {
 	@Override
 	public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
```

Comment:
Sorry, this whole method still has indentation issue. Please have another look. Thanks.

Reply:
Done.